### ISSUE TITLE
Fatal error: 'drawHierarchyInKeyWindow' requires tests to be run in a host application

### ISSUE THREAD

**Issue by jibeex** [pointfreeco/swift-snapshot-testing#377]
I am not sure why. The test target is run with a hosting app. But I am still getting this error. I am testing on X-Code 11.5. 

`sharedIfAvailable` is nil even `UIApplication.shared != nil`

```
static var sharedIfAvailable: UIApplication? {
      let sharedSelector = NSSelectorFromString("shared")
      guard UIApplication.responds(to: sharedSelector) else {
        return nil
      }
      
      let shared = UIApplication.perform(sharedSelector)
      return shared?.takeUnretainedValue() as! UIApplication?
  }
```

**Comment by ldstreet** [comment]
Seeing the same issue here.

**Comment by ldstreet** [comment]
Looks like this is a duplicate and has been solved in main. https://github.com/pointfreeco/swift-snapshot-testing/issues/365

**Comment by stephencelis** [comment]
We just released 1.8.2, so this should be resolved :smile:
